src/writeData/subscriptions/components/SubscriptionForm.tsx

```diff
@@ -96,7 +96,7 @@ const SubscriptionForm: FC<Props> = ({
               currentSubscription={formContent}
               updateForm={updateForm}
               className="create"
-              edit={false}
+              edit={true}
```

Comment:
edit is on by default?

Reply:
the same components are used for create and edit, so this one is a create instance so always editable

Reply:
ah, got it!